django_freeradius/api/views.py

```diff
@@ -137,3 +139,81 @@ def _get_status_type(self, request):
 
 
 accounting = AccountingView.as_view()
+
+
+class BatchPrefixView(generics.CreateAPIView):
+
+    queryset = RadiusBatch.objects.all()
+    serializer_class = RadiusBatchPrefixSerializer
+
+    def post(self, request, *args, **kwargs):
+        serializer = RadiusBatchPrefixSerializer(data=request.data)
+        if serializer.is_valid():
+            name = serializer.data.get('name')
+            strategy = self._get_strategy(request)
+            expiration_date = serializer.data.get('expiration_data')
+            prefix = serializer.data.get('prefix')
+            batch = RadiusBatch(name=name, strategy=strategy, expiration_date=expiration_date, prefix=prefix)
```

Comment:
are you sure you need to create this instance here?
the serializer should be able to save the object and return the instance, please check this, we want to use DRF properly and take advantage of its well tested and expected behaviour

Reply:
I did this because I want to create the users associated to the RadiusBatch, and I can't do this without this instance. I will check if I can do this without the instance but this is the only way that I found.